blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -172,7 +172,7 @@ private void buildProcessingChains() {
 
         this.defaultProcessorChain = AsyncChain
                 .withFunction(typeAndUnitProcessor)
-                .withFunction(rollupTypeCacher)
+                //.withFunction(rollupTypeCacher)
                 .withFunction(batchSplitter)
```

Comment:
Why is this line commented out?  Are you getting rid of the rollupTypeCacher too?